docs/DeveloperGuide.md

```diff
@@ -639,13 +639,28 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Finding a person
+
+1. Finding with parameters missing.
```

Comment:
Should this be compulsory parameters instead of just parameters

Reply:
no parameters are compulsory for `find` everything is optional using the term `compulsory parameter` will be misguiding, but there has to be some params